src/ansys/fluent/core/file_session.py

```diff
@@ -108,15 +121,17 @@ def add_scalar_fields_request(
 
         Raises
         ------
-        RuntimeError
+        SurfaceNameIDsProvided
             If surface names or surface ids are not provided.
-        RuntimeError
+        InvalidFieldNamePrefix
             If field name does not have prefix ``phase-`` for multi-phase cases.
         """
         if surface_ids is None:
             surface_ids = []
         if surface_ids and surface_names:
-            raise RuntimeError("Please provide either surface names or surface ids.")
+            raise SurfaceNameIDsProvided(
+                "Please provide either surface names or surface ids."
+            )
```

Comment:
It's a custom exception type, but the caller is providing the whole message, which makes the type practically redundant. Also the name of the type fails to describe the error.

Reply:
We will update all general error information.

Reply:
Are there some changes still to push

Reply:
yes we are going to define this message at single place only.

Reply:
Done. Thank you.